lib/ansible/modules/source_control/git.py

```diff
@@ -690,7 +690,7 @@ def get_head_branch(git_path, module, dest, remote, bare=False):
         module.fail_json(
             msg='Current repo does not have a valid reference to a '
             'separate Git dir or it refers to the invalid path',
-            details=str(err),
+            details=to_text(err),
```

Comment:
Module side usually uses to_native.  It's okay to use to_text with fail_json/exit_json, though, as those particular functions can handle string init as either bytes or text. Just letting you know that there's nothing wrong with setting to_native in modules instead.

Reply:
:+1: